otelcollector/shared/configmap/mp/configmapparser.go

```diff
@@ -140,13 +140,38 @@ func handleEnvFileError(filename string) {
 func Configmapparser() {
 	setConfigFileVersionEnv()
 	setConfigSchemaVersionEnv()
-	parseSettingsForPodAnnotations()
-	parsePrometheusCollectorConfig()
-	parseDefaultScrapeSettings()
-	parseDebugModeSettings()
 
-	tomlparserTargetsMetricsKeepList()
-	tomlparserScrapeInterval()
+	var metricsConfigBySection map[string]map[string]string
+	var err error
+	if os.Getenv("AZMON_AGENT_CFG_SCHEMA_VERSION") == "v2" {
```

Comment:
may be add this dimension as "v1/v2" to one of the existing metrics, for telemetry ?